src/JestExt.ts

```diff
@@ -259,7 +261,8 @@ export class JestExt {
     this.workspace.pathToJest = pathToJest(updatedSettings)
     this.workspace.pathToConfig = pathToConfig(updatedSettings)
 
-    this.jestSettings = new Settings(this.workspace)
+    const useShell = platform() === 'win32'
+    this.jestSettings = new Settings(this.workspace, { shell: useShell })
 

```

Comment:
Seeing the same thing twice makes me think we should move this into one spot:
```ts
updateSettings(workspace) {
  this.jestSettings = new Settings(workspace, { shell: platform() === 'win32' })
}
```

Reply:
I at first just thought of moving the settings out into its own property (a la `this.settingsOptions = { shell: platform() === 'win32' }`, but your approach is tidier than that.